apps/web/components/molecules/challengeItem/challengeItem.tsx

```diff
@@ -0,0 +1,42 @@
+import { Card } from 'atoms/card/card';
+
+type ChallengeItem = {
+	title: string;
+	description: string;
+	designs: string;
+	difficulty: string;
+	image: string;
+};
+
```

Comment:
Is `ChallengeItem` the same as `Challenge`? Maybe it would be better to have some common type in a shared directory, WDYT?

Reply:
Yes, i thought about folder and file with export types, but authors haven't created such, that's why i used this temp solution.

Reply:
https://github.com/Frontlive/Start-Coding/pull/31/commits/86058b99891d284e2e5b156168d06767bd63e821#diff-afc86f805a7994a923628e95fd646765c12ea9f598e2814d2e0422886d9fbe21